hack/run_test

```diff
@@ -372,15 +376,19 @@ function run_master_restart_test() {
   local PASS=master
 
   echo "Testing failed master restart"
-  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
   local cid_suffix="mrestart"
+  local network=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
~~This is tricky, in this context we're shadowing the global `network` with some random name.~~

Oh no, I see this is the same case as `docker_args`: declared `local` but accessed as a global from functions called from `run_master_restart_test`...

Might be just me, but I find that very surprising and confusing.

Possibly we already do that same here with other "local" variables?